pkg/encryption/encryption.go

```diff
@@ -127,7 +128,7 @@ func tryGetEncryptionKeyFromMetadataItem(namespace string, item v1alpha1.Metadat
 		return Key{}, errors.Errorf("%s: secretKeyRef cannot be empty", errPrefix)
 	}
 
-	r, err := secretStore.GetSecret(secretstores.GetSecretRequest{
+	r, err := secretStore.GetSecret(context.Background(), secretstores.GetSecretRequest{
```

Comment:
Please use `context.TODO()` here and/or add a comment `TODO: cascade context`

Reply:
Fixed.